### ISSUE TITLE
ContentService

### ISSUE THREAD

**Issue by SolomonShorser-OICR** [reactome/container#20]
Need to get this application: https://github.com/reactome/content-service
and this one: https://github.com/reactome/data-content
up and running.

But first, the Solr and Neo4j containers need to be set up.

Pre-requisites:
https://github.com/reactome/container/issues/15
https://github.com/reactome/container/issues/14

**Comment by AyushyaChitransh** [comment]
#14 Run Neo4j in container Completed! Working on #15 Solr Container

**Comment by SolomonShorser-OICR** [comment]
See the branch/ContentService: https://github.com/reactome/container/tree/feature/ContentService
I've added some code to build the java applications, but of course I have not yet tested it (there are probably still some bugs that I missed since I couldn't actually run the applications).

Once the work on solr is complete, please let me know and I will review and try to merge into feature/ContentService, then you will be able to test this item and finish anything else that needs work on it.

**Comment by AyushyaChitransh** [comment]
Content service is running and now I am working on data-content.

Status: Data-content available at http://localhost:8082/content/schema/DatabaseObject 
http://localhost:8082/content/ shows "The page requested is unavailable".

**Comment by SolomonShorser-OICR** [comment]
The data-content (AKA "content") application is never accessed at its root (`/content/`) because there is nothing there, so it will not return anything, so the "page is unavailable" is OK, I think.

I'll have a look and merge if everything is OK.

**Comment by AyushyaChitransh** [comment]
If that is the case then probably data-content is ready now. Please let me know if there are any problems.

**Comment by SolomonShorser-OICR** [comment]
Still some issues with getting solr search results to be displayed, but I think that is a separate issue.
